__init__.py

```diff
@@ -44,6 +43,9 @@ def __init__(self):
         super(DirectorySkill, self).__init__(name="DirectorySkill")
         self.url = "https://www.alamoanacenter.com/en/directory/"
 
```

Comment:
Override `self._speak_timeout = 60` or some value that should account for the longest combined speech output that comes before getting a user's response. Current behavior works for most cases, but not when there are several spoken locations that are closed